api/v1beta1/conditions.go

```diff
@@ -26,4 +28,12 @@ const (
 	WatcherAPIReadyErrorMessage = "WatcherAPI error occured %s"
 	// WatcherPrometheusSecretErrorMessage -
 	WatcherPrometheusSecretErrorMessage = "Error with prometheus config secret"
+	// WatcherApplierReadyInitMessage -
+	WatcherApplierReadyInitMessage = "WatcherApplier creation not started"
+	// WatcherAPIReadyRunningMessage -
```

Comment:
s/WatcherAPIReadyRunningMessage/WatcherApplierReadyRunningMessage and same with the other comments below

Reply:
done